experimental/LieAlgebras/test/WeylGroup-test.jl

```diff
@@ -1,5 +1,10 @@
 @testset "LieAlgebras.WeylGroup" begin
-  @testset "weyl_group(cartan_matrix::ZZMatrix)" begin
+  b3_w0 = UInt8[3, 2, 3, 1, 2, 3, 1, 2, 1]
```

Comment:
can you add a TODO comment somewhere in this file to remind us to enable groups conformance tests once https://github.com/Nemocas/AbstractAlgebra.jl/pull/1528 has propagated to OSCAR